src/components/CheckboxGroup/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { forwardRef } from "react";
+import useIsDeprecatedForm from "../../hooks/internal/useIsDeprecatedForm";
+import { ComponentWithStaticProperties } from "../../types";
+import { ComponentNames } from "../componentNames";
+import { defaultCheckboxGroupProps } from "./defaultCheckboxGroupProps";
+import { default as CheckboxGroupInternal } from "./CheckboxGroup";
+import { default as CheckboxGroupDeprecated } from "./CheckboxGroup_deprecated";
```

Comment:
might be better to disable default same as LSC :D

Reply:
That's what misha has throughout basis so won't do it until he comes back